### ISSUE TITLE
Training loss function of DenoisingDiT

### ISSUE THREAD

**Issue by lizi85** [BoyuanJiang/FitDiT#11]
Thank you for your work and for providing the inference code.

In the paper you state that "... the overall training loss function of DenoisingDiT is a combination of the imaged-based frequency loss and the latent-based diffusion noise prediction loss". 

Is this combination a simple sum, or a weighted sum of the two losses? Can you please provide some more insights regarding the training loss of the DenoisingDiT?

**Comment by BoyuanJiang** [comment]
We use the weighted sum form to make the two losses consistent in magnitude.
